src/main.py

```diff
@@ -21,10 +23,6 @@ async def on_ready(self) -> None:
             self.synced = True
         print(f"\033[32mBot is ready! {self.user}\033[0m")
 
-    async def on_message(self, message: discord.Message) -> None:
```

Comment:
What is the reason for removing this?

Reply:
whoops i was removing some lines when extracting only the wiktionary changes from the faq and must have accidentally removed it. my bad